src/main/java/tutorease/address/model/person/Role.java

```diff
@@ -21,7 +23,11 @@ public class Role {
     public Role(String role) {
         requireNonNull(role);
         checkArgument(isValidRole(role), MESSAGE_CONSTRAINTS);
-        value = role;
+        if (GUARDIAN_LOWERCASE.equals(role.toLowerCase())) {
+            value = GUARDIAN;
+        } else {
+            value = STUDENT;
+        }
```

Comment:
Maybe we can do if(guardian condition), else if (guardian condition) then else will throw an exception as it is neither student nor guardian.